llm/ext_server.go

```diff
@@ -0,0 +1,428 @@
+package llm
+
+/*
+#cgo CFLAGS: -I${SRCDIR}/llama.cpp/gguf -I${SRCDIR}/llama.cpp/gguf/common -I${SRCDIR}/llama.cpp/gguf/examples/server
+#cgo CFLAGS: -DNDEBUG -DLLAMA_SERVER_LIBRARY=1 -D_XOPEN_SOURCE=600 -DACCELERATE_NEW_LAPACK -DACCELERATE_LAPACK_ILP64
```

Comment:
I think we want to set `-DSERVER_VERBOSE=0` since it's currently outputting each token in the logs

Reply:
I'll follow up with a secondary PR to quiet down the server verbosity.